learn/advanced/known_limitations.md

```diff
@@ -57,6 +57,12 @@ If your query is `Hello - World`:
 
 **Explanation:** This limit is enforced for performance and storage reasons. Overly large internal data structures—resulting from documents with too many fields—lead to overly large databases on disk, and slower search performance.
 
+## Maximum number of concurrent requests
+
+**Limitation:** Meilisearch can receive a maximum of 126 API requests at the same time. Requests exceeding this limit will cause Meilisearch to thrown an internal error.
+
+**Explanation:** This limit applies only to receiving requests, not to processing or adding them to the Tasks queue. It is set by [LMDB](/learn/advanced/storage.md#lmdb), Meilisearch's storage engine.
```

Comment:
It does apply to the write calls, too, if you are trying to send more than 126 HTTP calls at the exact same time (quite complex) this error can be triggered.

Reply:
Apologies for my ignorance, but what would be the difference between a request and a write call in this case? Is there a term we can use to refer to both things at the same time?

Reply:
I was a little bit too picky here, you can keep what you wrote. I was referring that write operations (like inserting tasks) can also throw errors, but it is, in practice impossible, to do so.